web/geo/update-world.js

```diff
@@ -11,6 +11,7 @@ const config = {
   MIN_AREA_HOLES: 600000000,
   MAX_CONVEX_DEVIATION: 0.708,
   MIN_AREA_INTERSECTION: 500000,
+  SLIVER_RATIO: 0.01, // ratio of length and area to determine if the polygon is a sliver and should be ignored
```

Comment:
Hey @VIKTORVAV99,

I'm looking to use your sliver checking function.

One thing I noticed is area is returned in m^2 and line length in km, I think we should increase the sliver ratio to compensate. From my testing a ratio of 10 successfully identifies all of the 6 slivers I have in my data.

Below is an example of two sliver ratios for a thin and square polygon. I'm not sure exactly how line length is calculated by turf but my example assumes perimeter.

2000m x 0.5m  
Area = 1000m^2

Length = 2.001km

Sliver ratio: 1000m^2 / 2 = 500 m^2/km


1000m x 1000m

Area = 1,000,000m^2

Length = 4km

Sliver ratio: 250,000m^2/km

I think a sliver ratio of 10 is a nice conservative approach here but I'd appreciate your feedback.

Reply:
Oh man, you'll have to talk to @Kongkille about all this. He did the geographies and related changes and I haven't had time to dig into the details.